docs/.vuepress/public/_redirects

```diff
@@ -1,5 +1,70 @@
 # all redirects use 301 unless otherwise specified
 
+# LOTUS DOCS REDIRECTS
+# --------------------
+/get-started/store-and-retrieve/ https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
+/get-started/lotus/running-in-the-cloud https://lotus.filecoin.io/docs/set-up/running-in-the-cloud/ 
+/get-started/lotus/installation https://lotus.filecoin.io/docs/set-up/install/
+/get-started/lotus/switch-networks/#clean-rebuild-reinstall https://lotus.filecoin.io/docs/set-up/switch-networks/
+/get-started/lotus/chain/ https://lotus.filecoin.io/docs/set-up/chain-management/
+/get-started/lotus/send-and-receive-fil/ https://lotus.filecoin.io/docs/set-up/manage-fil/
+/get-started/lotus/multisig/ https://lotus.filecoin.io/docs/set-up/multisig/
+/get-started/lotus/ledger/#setup-your-ledger-device https://lotus.filecoin.io/docs/set-up/manage-fil/#ledger
+/get-started/lotus/upgrades/ https://lotus.filecoin.io/docs/set-up/upgrade/
+/get-started/lotus/tips-running-in-china/ https://lotus.filecoin.io/docs/set-up/nodes-in-china/
+/get-started/lotus/backup-and-restore/ https://lotus.filecoin.io/docs/set-up/backup-and-restore/
+/get-started/lotus/configuration-and-advanced-usage/#configuration https://lotus.filecoin.io/docs/set-up/configuration/
+/get-started/lotus/troubleshooting/ https://lotus.filecoin.io/docs/set-up/troubleshooting/
```

Comment:
```suggestion
/get-started/store-and-retrieve* https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
/get-started/lotus/running-in-the-cloud* https://lotus.filecoin.io/docs/set-up/running-in-the-cloud/ 
/get-started/lotus/installation* https://lotus.filecoin.io/docs/set-up/install/
/get-started/lotus/switch-networks* https://lotus.filecoin.io/docs/set-up/switch-networks/
/get-started/lotus/chain* https://lotus.filecoin.io/docs/set-up/chain-management/
/get-started/lotus/send-and-receive-fil* https://lotus.filecoin.io/docs/set-up/manage-fil/
/get-started/lotus/multisig* https://lotus.filecoin.io/docs/set-up/multisig/
/get-started/lotus/ledger* https://lotus.filecoin.io/docs/set-up/manage-fil/#ledger
/get-started/lotus/upgrades* https://lotus.filecoin.io/docs/set-up/upgrade/
/get-started/lotus/tips-running-in-china* https://lotus.filecoin.io/docs/set-up/nodes-in-china/
/get-started/lotus/backup-and-restore* https://lotus.filecoin.io/docs/set-up/backup-and-restore/
/get-started/lotus/configuration-and-advanced-usage* https://lotus.filecoin.io/docs/set-up/configuration/
/get-started/lotus/troubleshooting* https://lotus.filecoin.io/docs/set-up/troubleshooting/
```

I'm aware I shouldn't be testing in GitHub, but I'm on Windows and WSL isn't playing nice today...